cfpb/consumerfinance.gov
# Adding new redirect for /innovation/

Today (May 24), an update to the website will change the slug for /innovation/ to /competition-innovation/. This pull request adds a line to the redirect.conf file that should capture that page **and all subpages** and redirect them properly.


---




## Additions

- New line in redirects.conf


## How to test this PR

1. localhost
2. Change /innovation's slug to /competition-innovation and publish
3. Check that /innovation and subpage links now redirect properly


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
